charts/scalardl-auditor-monitoring/templates/_alloy-content.tpl

```diff
@@ -0,0 +1,99 @@
+{{/*
+Grafana Alloy components configuration
+*/}}
```

Comment:
This `_alloy-conten.tpl` file includes the whole configuration for Alloy. And, there are condition branches `{{- if .Values.global.envoy.enabled }}` to control whether to apply the envoy configurations or not.